### PR TITLE
Refactor the generate tarball logics

### PR DESCRIPTION
**Pull request by LuQQiu**
Refactor the generate tarball script

### CODE REVIEW

File: dev/scripts/src/alluxio.org/build-distribution/cmd/common.go [hunk 1]
```diff
@@ -51,41 +51,41 @@ type module struct {
...
+	"ufs-hadoop-3.2":  {"hadoop-3.2", "hdfs", true, "-pl underfs/hdfs -Pufs-hadoop-3 -Dufs.hadoop.version=3.2.1 -PhdfsActiveSync"},
+	"ufs-hadoop-3.3":  {"hadoop-3.3", "hdfs", false, "-pl underfs/hdfs -Pufs-hadoop-3 -Dufs.hadoop.version=3.3.1 -PhdfsActiveSync"},
+
 	"ufs-hadoop-ozone-1.2.1": {"hadoop-ozone-1.2.1", "ozone", true, "-pl underfs/ozone -Pufs-hadoop-3 -Dufs.ozone.version=1.2.1"},
 }
 
 var libJars = map[string]struct{}{
 	"integration-tools-hms":        {},
 	"integration-tools-validation": {},
-	"table-server-underdb-glue":    {},
```
Comment (Xenorith) [inline dev/scripts/src/alluxio.org/build-distribution/cmd/common.go:73]: confirm that we are removing these two table-server jars on purpose in this PR?

**Review by Xenorith — APPROVED** [review]

File: dev/scripts/src/alluxio.org/build-distribution/cmd/common.go (hunk 1 above)
Comment (LuQQiu) [reply dev/scripts/src/alluxio.org/build-distribution/cmd/common.go:73]: Yeah they need to be removed, i can separate PR if clearer

Comment (LuQQiu) [reply dev/scripts/src/alluxio.org/build-distribution/cmd/common.go:73]: @apc999 PTAL to confirm we remove the table server libs

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**